content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-05-01 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+# XMPP@home - 3 June 2020
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of May 2020.
+
+We started drafting every new issue in the [XSF Github repository](https://github.com/xsf/xmpp.org/pulls). 
+And we are always happy to welcome contributors. Just join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. 
+
+You have a project and (want to) write about it? Please consider sharing your news or events here, too!
+Even if you can only spend a few minutes, these would already be helpful.
+
+Tasks which need to be done on a regular basis are for example:
+
+- Aggregation of news in the XMPP universe
+- Short formulation of news and events
+- Summary of the monthly communication on extensions (XEP)
+- Review of the draft
+- Translations: especially French, German and Spanish
+
+The drafting process for each newsletter [is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+[Subscribe](https://tinyletter.com/xmpp) to receive the next edition in your inbox!
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here:
+
+- [German](https://www.jabber.de/)
+- [French](https://linuxfr.org/tags/xmpp/public)
+- [Spanish](https://www.jabber.de/)
+
+## Software releases
+
+It has been almost a year since the release of Gajim 1.1.3. A year of developing new features, cleaning up old code, and fixing bugs. This month, it is finally the time for a [first beta release of Gajim 1.2](https://gajim.org/post/2020-05-08-gajim-1.2.0-beta1-released/). Highlights are (amongst others): improved group chat system, completely rewritten network code, and a new account creation assistant. But there is much more to discover.
+
```

Comment:
Log in:
![Gajim 1.2.0-beta1 log in](https://gajim.org/img/news/account-creation-1.png)

Sign up:
![Gajim 1.2.0-beta1 sign up](https://gajim.org/img/news/account-creation-2.png)

Create account:
![Gajim 1.2.0-beta1 create account](https://gajim.org/img/news/account-creation-3.png)